src/frontend/apps/desk/src/pages/teams/create.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Input, Loader } from '@openfun/cunningham-react';
+import { ReactElement, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import IconGroup from '@/assets/icons/icon-group2.svg';
+import { Box, StyledLink, Text, WhiteBox } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+import { useCreateTeam } from '@/features/teams';
+import { NextPageWithLayout } from '@/types/next';
+
+import TeamLayout from './TeamLayout';
+
+const Page: NextPageWithLayout = () => {
+  const { t } = useTranslation();
+  const { mutate: createTeam, isError, isPending } = useCreateTeam();
+  const [teamName, setTeamName] = useState('');
+  const { colorsTokens } = useCunninghamTheme();
+
+  return (
+    <Box className="p-l" $justify="center" $align="start" $height="inherit">
+      <WhiteBox
```

Comment:
Should we name this one a `Card`?

Reply:
[30e0c2f](https://github.com/numerique-gouv/people/pull/43/commits/30e0c2fb85973c4fa090ec739429b1535193d1a2)